packages/gatsby/src/db/__tests__/nodes.js

```diff
@@ -0,0 +1,347 @@
+const { actions } = require(`../../redux/actions`)
+const { getNode, getNodes } = require(`../nodes`)
+const { store } = require(`../../redux`)
+require(`./fixtures/ensure-loki`)()
+
+describe(`stuff`, () => {
```

Comment:
`stuff` :D 

What's the reason for splitting part of `redux/__tests__/nodes.js` here? I don't quite understand that part

Reply:
Haha, can you tell I spent many hours in testing land? :P. I'll fix this up.

As for splitting `redux/__tests__/nodes.js`, half those tests were testing the `actions.js` functions as pure functions. I.e checking that the action and reducer result in specific states. The tests I moved into `db/__tests__/nodes.js` are testing all the way down to the database layer, regardless of the db implementation. This allows us to reuse as many tests as possible for both redux and loki.